### PR TITLE
Bugfix/attri2vec demo

### PR DESCRIPTION
**Pull request by daokunzhang**
I fixed the following bugs for attri2vec demos:

1. Specify the edge type as `cites` for attri2vec node classification and embedding learning demo.

2. Change the use_multiprocessing option of keras.model.fit_generator to False for attri2vec link prediction demo.

@adocherty, Andrew, could you please help me review it?

Best regards!
Daokun

### CODE REVIEW

File: demos/link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb [hunk 1]
```diff
@@ -533,17 +527,17 @@
...
-      "4711/4711 - 79s - loss: 0.4572 - binary_accuracy: 0.7603\n",
+      "4711/4711 - 201s - loss: 0.5845 - binary_accuracy: 0.6375\n",
       "Epoch 4/6\n",
-      "4711/4711 - 78s - loss: 0.2344 - binary_accuracy: 0.8970\n",
+      "4711/4711 - 195s - loss: 0.5185 - binary_accuracy: 0.7198\n",
       "Epoch 5/6\n",
-      "4711/4711 - 79s - loss: 0.0893 - binary_accuracy: 0.9697\n",
+      "4711/4711 - 194s - loss: 0.4662 - binary_accuracy: 0.7767\n",
       "Epoch 6/6\n",
-      "4711/4711 - 79s - loss: 0.0321 - binary_accuracy: 0.9932\n"
```
Comment (adocherty) [inline demos/link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb]: The accuracy has gone down a lot - is this expected?

File: demos/link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb [hunk 2]
```diff
@@ -552,8 +546,8 @@
     "    generator.flow(unsupervised_samples),\n",
     "    epochs=epochs,\n",
     "    verbose=2,\n",
-    "    use_multiprocessing=True,\n",
-    "    workers=4,\n",
+    "    use_multiprocessing=False,\n",
+    "    workers=1,\n",
```
Comment (adocherty) [inline demos/link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb]: Do you need `workers=1` if you are turning multiprocessing off?

**Comment by daokunzhang** [comment]
Thanks @adocherty !

**Comment by daokunzhang** [comment]
@adocherty, Andrew, should I do something to merge the pull request? like signing commits?
